visual-java/src/main/java/com/saucelabs/visual/graphql/CreateSnapshotFromWebDriverMutation.java

```diff
@@ -60,12 +63,14 @@ public CreateSnapshotFromWebDriverIn(
         String jobId,
         String name,
         String sessionId,
-        String sessionMetadata) {
+        String sessionMetadata,
+        List<IgnoreSelectorIn> ignoreSelectors) {
```

Comment:
Same here, we need to keep support of previous `CreateSnapshotFromWebDriverIn` constructor.